### PR TITLE
Module 5 components 2

### PR DESCRIPTION
**Pull request by Yurbly**

### CODE REVIEW

File: src/components/common/form/Select/types.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export interface IOption {
+    label: string;
+    value: string;
+}
+
+export type IsMulti = false;
```
Comment (nvkuznetsova) [inline src/components/common/form/Select/types.tsx]: Should this type be an alias of `boolean`?

File: src/components/modals/AddEditMovieModal/AddEditMovieModal.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+    const {titles, placeholders} = commonText.addEditModal;
+
+    return (
+        <ModalComponent
+            isOpen={isOpen}
+            contentStyle={contentStyle}
+            onClose={onClose}
+        >
+            <Title>Add movie</Title>
```
Comment (nvkuznetsova) [inline src/components/modals/AddEditMovieModal/AddEditMovieModal.tsx]: Please, change the modal title depending on 'add' or 'edit' mode

File: src/components/modals/AddEditMovieModal/AddEditMovieModal.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                    onChange={(value: string) => onInputFieldChange("overview", value)}
+                />
+            </Form>
+            <Controls>
+                <Button
+                    title={commonText.buttonTitles.reset}
+                    styleType={buttonTypes.cancel}
+                    onClick={useCallback(() => setMovieData(defaultMovieData), [])}
+                />
+                <Button title={commonText.buttonTitles.submit} onClick={() => console.log("submit")}/>
```
Comment (nvkuznetsova) [inline src/components/modals/AddEditMovieModal/AddEditMovieModal.tsx]: Please, close the model on submit

File: src/components/movie/list/MovieCard.tsx [hunk 4]
```diff
@@ -62,13 +73,36 @@ const MovieCard: FC<IMovieCardProps> = props => {
 
     const posterSrc = poster_path || movieMockImgUrl;
 
+    const menuItems = [
```
Comment (nvkuznetsova) [inline src/components/movie/list/MovieCard.tsx:79]: Let's move these items to constants

Comment (Yurbly) [reply src/components/movie/list/MovieCard.tsx:79]: I need functions from props here, it is very inconvenient to move whole array to consts. So I only turned string ids to local constants

**Pull request merged**